pdfmaker/css/holt/pdf.css

```diff
@@ -1,9 +1,1175 @@
+  @top-left {
+    content: flow(runheadleft);
+    font-family: "MinionPro";
+    font-style: italic;
+    font-size: 9pt;
+    vertical-align: top;
+    padding: 35pt 0 0 0;
+    margin: 0; 
+    text-align: left; } }
+@page front:right {
+  @top-right {
+    content: flow(runheadright);
+    font-family: "MinionPro";
+    font-style: italic;
+    font-size: 9pt;
+    vertical-align: top;
+    padding: 35pt 0 0 0;
+    margin: 0; 
+    text-align: right; } }
+@page front:first {
+  @top-right {
+  content: normal; }
+  @top-left {
+  content: normal; } }
+@page front:blank {
+  @top-right {
+  content: normal; }
+  @top-left {
+  content: normal; } }
+
+@page back:left {
+  @top-left {     
+    content: flow(runheadleft);   
+    font-family: "MinionPro";
+    font-style: italic;
+    font-size: 9pt;
+    vertical-align: top;
+    padding: 35pt 0 0 0;
+    margin: 0; 
+    text-align: left; } }
+@page back:right {
+  @top-right {
+    content: flow(runheadright);    
+    font-family: "MinionPro";
+    font-style: italic;
+    font-size: 9pt;
+    vertical-align: top;
+    padding: 35pt 0 0 0;
+    margin: 0; 
+    text-align: right; } }
+@page back:first {
+  @top-right {
+  content: normal; }
+  @top-left {
+  content: normal; } }
+@page back:blank {
+  @top-right {
+  content: normal; }
+  @top-left {
+  content: normal; } }
+
+body {
+  font-family: "MinionPro";
+  counter-reset: endnotes chapternumber;
+}
+
+section, p, div, h1, h2, h3, h4, h5, h6 {
+  display: block;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  page-break-after: avoid;
+}
+
+p {
+  font-size: 11.2pt;
+  line-height: 15pt;
+  text-indent: 15pt;
+  text-align: justify;
+  text-align-last: left;
+  margin: 0;
+  widows: 1;
+  orphans: 1;
+  hyphens: auto;
+  prince-hyphenate-after: 3;
+  prince-hyphenate-before: 3;
+  hyphenate-lines: 2;
+}
+
+/*run heads (in combination with holt.js)*/
+div.runheadright {
+  flow: static(runheadright);
+  font-family: "MinionPro";
+  font-style: italic;
+  font-size: 9pt;
+  text-align: right;
+  padding: 0;
+  margin: 0;
+  text-indent: 0; }  
+div.runheadleft {
+  flow: static(runheadleft);
+  font-family: "MinionPro";
+  font-style: italic;
+  font-size: 9pt;
+  text-align: left;
+  padding: 0;
+  margin: 0;
+  text-indent: 0; }  
+
+section[data-type="preface"] p + div.runheadright:after,
+nav[data-type="toc"] p + div.runheadright:after {
+content: " • " counter(page, lower-roman); }
+
+section[data-type="preface"] p + div.runheadright + div.runheadleft:before,
+nav[data-type="toc"] p + div.runheadright + div.runheadleft:before {
+content: counter(page, lower-roman) " • "; }
+
+section[data-type="chapter"] p + div.runheadright:after, 
+section[data-type="appendix"] p + div.runheadright:after {
+content: " • " counter(page); }
+
+section[data-type="chapter"] p + div.runheadright + div.runheadleft:before {
+content: counter(page) " • " string(title); }
+
+section[data-type="appendix"] p + div.runheadright + div.runheadleft:before {
```

Comment:
This part is right (the rest of the content is supplied by the JS) but this made me realize I had to edit the JS again - I had to insert empty divs for the left page chapter runheads :)